src/SDKs/Azure.ApplicationModel.Configuration/data-plane/Azure.ApplicationModel.Configuration/ConfigurationClient_private.cs

```diff
@@ -208,6 +208,18 @@ static ReadOnlyMemory<byte> Serialize(ConfigurationSetting setting)
             return content;
         }
 
+        static ReadOnlyMemory<byte> Serialize(string value)
+        {
+            ReadOnlyMemory<byte> content = default;
+            int size = 256;
+            byte[] buffer = new byte[size];
+            if (ConfigurationServiceSerializer.TrySerialize(value, buffer, out int written))
```

Comment:
If the value is greater than 256 this will fail to serialize the value and return a null content -- is that the desired behavior?